### DISCUSSION TITLE
Trivy local db upgradation

### DISCUSSION THREAD

**Discussion by gulammoideen** [aquasecurity/trivy#5156] [Q&A] [answered]
### Question

We have been using trivy scan for scanning the container images for so long in our client environment which is a restricted environment (Air Gapped Environment). A new requirement came to update the local offline trivy db. 
Please help me to update the trivy db version 2 to the latest version. 
 trivy
├── db
│   ├── metadata.json
│   └── trivy.db
└── fanal
    └── fanal.db

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Client/Server

### Operating System

Red Hat Enterprise Linux 8

### Version

```bash
[devops@rs-brm-cicd-201 trivy]$ trivy -v
Version: 0.30.4
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-08-23 12:06:51.578114346 +0000 UTC
  NextUpdate: 2022-08-23 18:06:51.578113946 +0000 UTC
  DownloadedAt: 2022-08-23 13:20:00.343446174 +0000 UTC
```

**Comment by knqyf263** [comment, marked as answer]
Please see https://aquasecurity.github.io/trivy/v0.45/docs/advanced/air-gap/
